src/main/java/ru/skypro/homework29/controller/EmployeeController.java

```diff
@@ -0,0 +1,39 @@
+package ru.skypro.homework29.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ru.skypro.homework29.Employee;
+import ru.skypro.homework29.service.EmployeeService;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/departments")
+public class EmployeeController {
```

Comment:
```suggestion
public class DepartmentController {
```
это новый контроллер, старый EmployeeController по-прежнему есть